packages/asset/contracts/AssetCreate.sol

```diff
@@ -354,5 +356,267 @@ contract AssetCreate is
         return ERC2771HandlerUpgradeable._msgData();
     }
 
+    bytes32 public constant LAZY_MINT_TYPEHASH =
+        keccak256(
+            "LazyMint(address creator,uint16 nonce,uint8 tier,uint256 amount,uint256 unitPrice,address paymentToken,string metadataHash,uint256 maxSupply)"
+        );
+
+    bytes32 public constant LAZY_MINT_BATCH_TYPEHASH =
+        keccak256(
+            "LazyMintBatch(address[] creators,uint16 nonce,uint8[] tiers,uint256[] amounts,uint256[] unitPrices,address[] paymentTokens,string[] metadataHashes,uint256[] maxSupplies)"
+        );
+
+    // A fee denoted in BPS that is charged when using the lazy minting feature, deducted from the creators payout
+    uint256 public lazyMintFee;
+    // The address that receives the lazy mint fee
+    address public lazyMintFeeReceiver;
+    // mapping of tokenId => maxSupply specified by the creator
+    mapping(uint256 => uint256) public availableToMint;
+
+    struct LazyMintData {
+        uint8 tier;
+        uint256 amount;
+        uint256 unitPrice;
+        address paymentToken;
+        string metadataHash;
+        uint256 maxSupply;
+        address creator;
+    }
+
+    struct LazyMintBatchData {
+        address[] creators;
+        uint8[] tiers;
+        uint256[] amounts;
+        uint256[] unitPrices;
+        address[] paymentTokens;
+        string[] metadataHashes;
+        uint256[] maxSupplies;
+    }
```

Comment:
Move new vars to above constructor 
Move structs into IAssetCreate ? (We have the AssetData struct in IAsset)